.appveyor.yml

```diff
@@ -4,7 +4,7 @@ image: Visual Studio 2017
 platform: x64
 
 # Don't clone the entire repo.
-clone_depth: 10
+clone_depth: 5
```

Comment:
Out of curiosity... why reduce the depth?

Reply:
Good question. I was trying to think of any other ways to help speed up the AppVeyor builds -- since they are now failing due to timeouts on a semi-regular basis. :(
By slight reducing the clone depth we can save a bit more time as the build doesn't need to clone down as much data.

Reply:
And does the appveyor checks need a certain depth? Like, why stop at 5?

Reply:
The number is actually somewhat arbitrary. Most of the time we only need the top most commit (ie: a depth of 1). The only time that we need to sync down more commits is if somebody wants to do a "re-build" on older commits. For example, if a build times out or fails, and somebody wants to kick off a rebuild -- then we'd need to sync down enough history to "reach" the older commit to rebuild.

Reply:
Ahhh I see, that's good to know 👍  Thank you Jeff :D

Reply:
No problem! :)